.travis.yml

```diff
@@ -37,7 +37,7 @@ script:
         cargo rustc --verbose --example ci_test -- -Zno-trans &&
         cargo rustc --verbose --example key_value_store -- -Zno-trans &&
         cargo rustc --verbose -- -Zno-trans &&
-        cargo test  --verbose --features use-mock-crust
+        env RUSTFLAGS="-C codegen-units=8" cargo test --release --verbose --features use-mock-crust --test mock_crust_tests
```

Comment:
Ah, it looks like running the lib tests requires compiling without `--crate_type lib`, so `src/lib.rs` does get compiled twice...